app/services/encryption/regional_digest_pair.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+Encryption::RegionalDigestPair = RedactedStruct.new(
```

Comment:
What's the reasoning for changing the name here to "digest"?

I think the way that we use this class neither "digest" nor "ciphertext" is technically correct. Within password encryption we use a hash function at which point we create a digest which is not reversible. The PII encryption however creates a reversible ciphertext.

Maybe we could change this to something like `RegionalEncryptedValue`?

Reply:
Yeah, that makes sense. I've renamed in 8579f09f95.